### DISCUSSION TITLE
Whats the best way to keep a DIV in the same position?

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#682] [Answers & Questions]
Yeah, so when I try to position a DIV in Cables, when it is position absolute or fixed, if the canvas gets resize, it doesn't stay in a relative position.  What would be the best way to do that?
<img width="572" alt="Screenshot 2024-02-08 at 13 45 05" src="https://github.com/cables-gl/cables_docs/assets/12829050/be64d6ba-3042-4959-8c93-689006fca9aa">

And why is px and % in `Op.ElementCssTransform`  difficult to understand? Seems like % doesn't really compute against the canvas size?

**Comment by cristianvogel** [comment]
Self answer:

use ```TransformCSS3DElement``` with Op.Transform . Nice

**Comment by pandrr** [comment]
this is just using normal css/html 
https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate

not sure what same "position means", by default they are positioned from the top left corner, thats the same position for me.

if you want it to be the same position as szome 3d element use transformelement op

or use other browser css features to position them
